### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by dbason**

### CODE REVIEW

File: pkg/metrics/collector/remote.pb.go [hunk 1]
```diff
@@ -179,39 +179,40 @@ var file_pkg_metrics_collector_remote_proto_rawDesc = []byte{
...
-	0x2e, 0x44, 0x65, 0x73, 0x63, 0x72, 0x69, 0x70, 0x74, 0x6f, 0x72, 0x4c, 0x69, 0x73, 0x74, 0x22,
-	0x00, 0x28, 0x00, 0x30, 0x00, 0x12, 0x3e, 0x0a, 0x07, 0x43, 0x6f, 0x6c, 0x6c, 0x65, 0x63, 0x74,
-	0x12, 0x16, 0x2e, 0x67, 0x6f, 0x6f, 0x67, 0x6c, 0x65, 0x2e, 0x70, 0x72, 0x6f, 0x74, 0x6f, 0x62,
-	0x75, 0x66, 0x2e, 0x45, 0x6d, 0x70, 0x74, 0x79, 0x1a, 0x15, 0x2e, 0x63, 0x6f, 0x6c, 0x6c, 0x65,
-	0x63, 0x74, 0x6f, 0x72, 0x2e, 0x4d, 0x65, 0x74, 0x72, 0x69, 0x63, 0x4c, 0x69, 0x73, 0x74, 0x22,
-	0x00, 0x28, 0x00, 0x30, 0x00, 0x1a, 0x00, 0x42, 0x2f, 0x5a, 0x2d, 0x67, 0x69, 0x74, 0x68, 0x75,
-	0x62, 0x2e, 0x63, 0x6f, 0x6d, 0x2f, 0x72, 0x61, 0x6e, 0x63, 0x68, 0x65, 0x72, 0x2f, 0x6f, 0x70,
-	0x6e, 0x69, 0x2f, 0x70, 0x6b, 0x67, 0x2f, 0x6d, 0x65, 0x74, 0x72, 0x69, 0x63, 0x73, 0x2f, 0x63,
-	0x6f, 0x6c, 0x6c, 0x65, 0x63, 0x74, 0x6f, 0x72, 0x62, 0x06, 0x70, 0x72, 0x6f, 0x74, 0x6f, 0x33,
+	0x2f, 0x65, 0x6d, 0x70, 0x74, 0x79, 0x2e, 0x70, 0x72, 0x6f, 0x74, 0x6f, 0x1a, 0x4f, 0x2f, 0x68,
```
Comment (kralicky) [inline pkg/metrics/collector/remote.pb.go:182]: what caused this change?

Comment (dbason) [reply pkg/metrics/collector/remote.pb.go:182]: I'm not sure, I just ran generate and it spit it out.

File: apis/v1beta2/monitoring_types.go [hunk 2]
```diff
@@ -43,6 +43,7 @@ type GatewaySpec struct {
 	//+kubebuilder:validation:Required
 	Auth             AuthSpec `json:"auth,omitempty"`
 	Hostname         string   `json:"hostname,omitempty"`
+	GrafanaHostname  string   `json:"grafanaHostname,omitempty"`
```
Comment (kralicky) [inline apis/v1beta2/monitoring_types.go:46]: This works but let's move it to NoauthSpec

Comment (kralicky) [inline apis/v1beta2/monitoring_types.go:46]: This works but let's move it to NoauthSpec

**Review by kralicky — CHANGES_REQUESTED** [review]

**Pull request merged**